packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -2,21 +2,19 @@
 
 ## Next
 
+## 0.1.2
+
 - Add `help` prop support for `SelectControl` used in the `select` DataForm control, via the DataForm field `description` prop.
 - Add a new DataForm Edit control: `toggleGroup`, which renders a `<ToggleGroupControl />`. If the field elements (options) have a `description`, then the selected option's description will be also rendered.
 - Implement the `media` field type definition and allow type definitions to provide a new default: `enableSorting`.
 - Add new `boolean` field type definition and edit control. Field type definitions are able to define a default render function that will be used if the field doesn't define one.
 - Pin the actions column on the table view when the width is insufficient.
-- Bring changes from @wordpress/dataviews 4.19.0 (no updates in this version).$
```

Comment:
This is missing in the changelog. It's important that it's always present, see PCYsg-173c-p2

Reply:
Oh, I assumed that if it says `(no updates in this version)`, we could leave it out of the release notes. Thanks for clarifying. I've added it back in.